MQTTClientGJ.podspec

```diff
@@ -5,37 +5,9 @@ Pod::Spec.new do |mqttc|
 	mqttc.homepage     = "https://github.com/gojek/courier-iOS"
 	mqttc.license      = "EPLv1"
 	mqttc.author       = { "Alfian Losari" => "alfian.losari@gojek.com" }
-	mqttc.source       = {
-		:git => "https://github.com/gojek/courier-iOS.git",
-		:tag => "#{mqttc.version}"
-	}
+	mqttc.source          = { http: "http://artifactory-gojek.golabs.io/artifactory/gojek-ios-pods/MQTTClientGJ/MQTTClientGJ_#{mqttc.version}.tar.gz" }
```

Comment:
We can't use private artifactory as MQTTClientGJ is distributed as public Cocoapods https://cocoapods.org/pods/MQTTClientGJ

Reply:
Basically, other people should be able to just add 
`pod 'MQTTClientGJ`` in their `Podfile` then run 'pod install`

Reply:
Changed this to point to source code, as we anyway are checking in the generated .xcframework so from source code the clinets should be able to find .xcframework.